frontend/src/index.js

```diff
@@ -1,12 +1,34 @@
-import React from 'react'
+import React, { Component } from 'react'
 import ReactDOM from 'react-dom'
-import { BrowserRouter as Router, Route } from 'react-router-dom'
-import Dashboard from './components/Dashboard.js'
+import { Provider } from 'react-redux'
+import { Route, withRouter } from 'react-router'
+import { ConnectedRouter } from 'react-router-redux'
+import configureStore, { history } from './redux/configureStore'
+import { Dashboard } from './components'
+
+const store = configureStore()
+
+class ScrollToTop extends Component {
+  componentDidUpdate(prevProps) {
+    window.scrollTo(0, 0)
+  }
+
+  render() {
+    return this.props.children
+  }
+}
+
+const TopScroll = withRouter(ScrollToTop)
 
 ReactDOM.render(
-  <Router onUpdate={() => window.scrollTo(0, 0)}>
-    <Route exact path="/" component={Dashboard} />
-  </Router>,
+  <Provider store={store}>
+    <ConnectedRouter onUpdate={() => window.scrollTo(0, 0)} history={history}>
+      <TopScroll>
+        <div className="heightDef">
+          <Route exact path="/" component={Dashboard} />
+        </div>
+      </TopScroll>
+    </ConnectedRouter>
```

Comment:
ConnectedRouter is available in`react-router-redux` in the [v5 release](https://www.npmjs.com/package/react-router-redux/v/5.0.0-alpha.8). This is an alpha version though, so it's ahead of the latest stable release. If you console log `ConnectedRouter` right now it shows up as `undefined` because it doesn't exist in v4. You can either switch back to `BrowserRouter` which you were using before, or switch to the v5 alpha version (I haven't had any problems with it). You can use the version I used with Child's Play in our [`package.json`](https://github.com/hack4impact-uiuc/childs-play-tool/blob/master/frontend/package.json). Just use the `react-router-redux` one and leave the rest unless you need to change them.